lib/utils/redux/redux_bundle.dart

```diff
@@ -59,11 +59,8 @@ class ReduxBundle {
         _notificationsService =
             notificationsService ?? NotificationsService(FirebaseMessaging()),
         _storageService = storageService ??
-            StorageService(
-              FirebaseStorage(
-                  app: FirebaseFirestore.instance.app,
-                  storageBucket: _bucketName),
-            ),
+            StorageService(FirebaseStorage.instanceFor(
+                app: FirebaseFirestore.instance.app, bucket: _bucketName)),
```

Comment:
this is the only change that wasn't folder renaming: just updates the declaration of a FirebaseStorage object to the new method.